### ISSUE TITLE
[Question] Legality of referencing the Kozuka fonts to recreate missing JP-style glyphs

### ISSUE THREAD

**Issue by Marcus98T** [adobe-fonts/source-han-sans#474]
As we know, there are quite a lot of [missing JP glyphs that are part of Adobe-Japan1](https://github.com/adobe-fonts/source-han-sans/issues/425), and that CN glyphs, which are designed differently from the JP glyphs, are being used instead. Because Adobe is slow in their updates due to different priorities and the immense difficulty in maintaining the Source Han/Noto fonts, I plan to recreate the missing JP glyphs myself, based off the Kozuka fonts, for better harmonisation of the font design in my fork.

It is a known fact that [a good amount of Source Han Sans kanji glyphs are based off Kozuka Gothic](https://blog.typekit.com/2014/08/14/interview-with-ryoko-nishizuka/), and for Serif, the kanji are based off Kozuka Mincho, but with slightly taller horizontal stems.

However, the Kozuka fonts are proprietary commercial fonts, and even if I remove the decorative serifs (for Sans), adjust the Source Han Sans/Serif glyphs to closely match the Kozuka font proportions (without copying the Kozuka font designs directly, just taking outlines from other JP glyphs in Source Han with similar components and adjusting them), and **not** use the Kozuka Gothic radical designs for which the JP glyphs in Source Han Sans are designed differently (e.g. the left 酉 radical, example image below), there is a chance that it can be deemed a copyright infringement.

<img width="686" alt="Screenshot 2023-11-04 at 16 11 53" src="https://github.com/adobe-fonts/source-han-sans/assets/10541683/11838f03-d99c-42d7-b721-01ce975e5a74">

Which is why I am asking in good faith, is it legally allowed to just reference the glyph shapes of the Kozuka fonts to recreate the missing JP-style glyphs? Or is it better that I wait for the [unreleased JP glyphs to go public as I mentioned before](https://github.com/adobe-fonts/source-han-sans/issues/292#issuecomment-1606146908)? I'd really want the latter to happen so it can be legally allowed under the Open Font License, but the way I see it, it is of very low priority for Adobe.

In other words, I am asking for permission to reference the Kozuka fonts as long as I keep the design of Source Han Sans/Serif and never ever take the designs of the Kozuka fonts.

While this issue is not an urgent matter (because I can choose not to release my fork publicly or just use the CN glyphs in the meantime), I think it is important. This issue can be moved to a discussion if needed.
